### PR TITLE
Docker user destination

### PR DESCRIPTION
**Pull request by bgruening**
@gmauro does this look good to you?

### CODE REVIEW

File: files/galaxy/dynamic_rules/usegalaxy/destination_specifications.yaml [hunk 1]
```diff
@@ -323,7 +323,39 @@ remote_cluster_mq_docker_de03:
...
+    rewrite_parameters: True
+    transport: 'curl'
+    # can and should be made stricter at some point
+    docker_volumes: '$galaxy_root:ro,$tool_directory:ro,$working_directory:rw,$job_directory:rw,/data/share/staging/:ro'
+    docker_memory: '{MEMORY}'
+    docker_auto_rm: true
+    docker_default_container_id: busybox:ubuntu-14.04
+    docker_sudo: false
+    docker_set_user: ''
+    docker_run_extra_arguments: '--gpus all'
```
Comment (gmauro) [inline files/galaxy/dynamic_rules/usegalaxy/destination_specifications.yaml:355]: I think you should put all the extra arguments together

File: files/galaxy/dynamic_rules/usegalaxy/destination_specifications.yaml [hunk 2]
```diff
@@ -323,7 +323,39 @@ remote_cluster_mq_docker_de03:
     docker_run_extra_arguments: '--gpus all'
     tmp_dir: 'True'
 
-
+remote_cluster_mq_docker_de03_user:
```
Comment (gmauro) [inline files/galaxy/dynamic_rules/usegalaxy/destination_specifications.yaml:326]: NO. Rename as remote_cluster_mq_docker_user_de03

File: files/galaxy/dynamic_rules/usegalaxy/tool_destinations.yaml [hunk 3]
```diff
@@ -23,7 +23,7 @@ xchem_pose_scoring:
       CUDA_VISIBLE_DEVICES: 0
 
 openduck_run_smd:
-   runner: remote_cluster_mq_docker_de03
+   runner: remote_cluster_mq_docker_de03_user
```
Comment (gmauro) [inline files/galaxy/dynamic_rules/usegalaxy/tool_destinations.yaml:26]: Rename as remote_cluster_mq_docker_user_de03

File: group_vars/galaxy.yml [hunk 4]
```diff
@@ -634,6 +634,25 @@ galaxy_jobconf:
               docker_sudo: 'false'
               docker_run_extra_arguments: '--gpus all'
               docker_set_user: ''
+        - id: remote_cluster_mq_docker_de03_user
```
Comment (gmauro) [inline group_vars/galaxy.yml:637]: NO. Rename as remote_cluster_mq_docker_user_de03

**Comment by gmauro** [comment]
and I guess you need to add an entry here:
https://github.com/usegalaxy-eu/infrastructure-playbook/blob/master/files/galaxy/config/user_preferences_extra_conf.yml

**Comment by bgruening** [comment]
> and I guess you need to add an entry here as well:
https://github.com/usegalaxy-eu/infrastructure-playbook/blob/master/files/galaxy/config/user_preferences_extra_conf.yml

This is only for one single tool, don't think that makes sense to add it here.

**Pull request closed without merging**